device_doctor/bin/main.dart

```diff
@@ -14,9 +15,15 @@ const List<String> supportedDeviceOS = <String>['ios', 'android'];
 
 /// These values will be initialized in `_checkArgs` function,
 /// and used in `main` function.
+String _action;
 String _deviceOS;
 
-/// Manage healthcheck and recovery for devices.
+/// Manage `healthcheck` and `recovery` for devices.
+///
+/// For `healthcheck`, if no device is found or any health check fails an stderr will be logged,
+/// and an exception will be thrown.
+///
+/// For `recovery`, device will do cleanup, reboot, and then dismiss any dialog for `ios` device.
```

Comment:
> then dismiss any dialog for `ios` device

Could this be generalized to initialize device? If there's future dialogs that need to be closed for android, this doc would fall out of date.

Reply:
Updated to make it more general.